dispatchr-ruby/app/controllers/request_items_controller.rb

```diff
@@ -0,0 +1,63 @@
+class RequestItemsController < ApplicationController
+  before_action :set_request_item, only: [:show, :update, :destroy]
+
+  # GET /request_items
+  # GET /request_items.json
+  def index
+    render json: RequestItem.all
+  end
+
+  # GET /request_items/1
+  # GET /request_items/1.json
+  def show
+    render json: @request_item
```

Comment:
I noticed in Alok's controller, he has
` def show
     address = Address.find(params[:id])
     render json: address
 end`

Where's @request_item set? and why is `.find(params[:id])` missing?